assets/js/coder.js

```diff
@@ -76,6 +76,9 @@ function setTheme(theme) {
         
     }
     
+    // Create and send event
+    const event = new Event('themeChanged');
+    document.dispatchEvent(event);
```

Comment:
I need to refactor every comment system integration to use events too. This look way nicer.